masonry_winit/examples/calc_masonry.rs

```diff
@@ -347,12 +345,17 @@ fn flex_row(
         .with_flex_child(w4, 1.0)
 }
 
-fn build_calc() -> impl Widget {
+#[derive(Default, Clone)]
+struct WidgetIds {
+    display: WidgetId,
+}
```

Comment:
This doesn't need to implement Default. You could add a `new()` or `create()` method.